src/app/lib/config/services/portuguese.js

```diff
@@ -23,7 +23,8 @@ export const service = {
     brandName: 'BBC News Brasil',
     product: 'BBC News',
     serviceLocalizedName: 'Brasil',
-    defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/brasil.png',
+    defaultImage:
+      'https://news.files.bbci.co.uk/include/articles/public/portuguese/images/og/poster-1024x576.png',
```

Comment:
This image is not just used for `og` opengraph, but also for `twitter:image` so I'd sugest naming the directory something like `/public/portuguese/images/metadata` instead.

Reply:
OK